### DISCUSSION TITLE
Webcam suddenly stop working during the test

### DISCUSSION THREAD

**Discussion by minh210999** [SafeExamBrowser/seb-win-refactoring#178] [Q&A]
Hi,

While I was doing my exam, the webcam suddenly stopped working on the submission step, therefore, I couldn't submit my work. I'm running windows 11 and the latest version of SEB. 

Is this normal behavior, is there anything I can do?

**Comment by dbuechel** [comment]
No, that is not normal behavior, but there might be a lot of reasons why that would happen which may not have anything to do with SEB itself. Was there any indication that the webcam stopped working due to some error in SEB itself?

**Reply by minh210999** [reply]
I think the problem lay on SEB. After posting, I've tried to recreate the issue again. It seems like the webcam will always turn off on the submission page. Moreover, if I refresh that page again, the webcam will turn back on, then I can submit my work as usual.

Is there any way we can tell the root of this issue?

**Reply by dbuechel** [reply]
What exam system are you using? Looking at it from afar, it seems more likely that there is an error on that page if it works seamlessly up to that point. You may find some further information in the browser log file(s), to be found under `%LocalAppData%\SafeExamBrowser\Logs`.
